### DISCUSSION TITLE
How can I use in only 1 user

### DISCUSSION THREAD

**Discussion by MuscularPuky** [valinet/ExplorerPatcher#2547] [Q&A]
ExplorerPatcher is installed locally so available in any users. but I want to use it in only 1 user profile. I just heard there's user specific config registry. Is there enable/disable key? or can I disable or prevent startup for only 1 user, without uninstall? or install for only current user??

**Comment by pyrates999** [comment]
EP does store its settings per user in the registry.  So maybe.  But it requires administrative privileges to install and update.

**Comment by MuscularPuky** [comment]
I couldn't find in "disable" or "enable" or such similar key in registry. so my idea: if I delete EP registry key, is it work in default prefereces? or make default keys? or stopped work for that 1 user?

**Reply by pyrates999** [reply]
Currently there is no such feature.
